citi-onboarding/Maracatec
# CRUD of Carousel was created

**What I Did**
------------------

-  The methods: create, gete, delete and update

**How to Test**
-------------------

1. Open the terminal and go to the folder of this project

2. In terminal:      
       i. `git checkout develop`
      ii. `git pull`
     iii. `git checkout feature/Carousel`
     iv. `git pull origin feature/Carousel`
      v. `open the folder server with comand cd server/`
      vi. `yarn install or npm install`
      vii. `With Docker opened, execute the comander docker-compose up`
      viii. `Open the Insomnia and test each one of methods. POST, GET, DELETE, and UPDATE`

3. Method POST:
    ![image](https://user-images.githubusercontent.com/91433976/170573192-400826ff-dbce-4dc5-87e3-ab479895d3b4.png)
    ![image](https://user-images.githubusercontent.com/91433976/170573297-7cb7da5c-2eec-45c7-814e-e53c1c377b1c.png)

4. Method GET:
   ![image](https://user-images.githubusercontent.com/91433976/170573460-e8304e9a-8264-4063-b2ca-7452e576b147.png)
   ![image](https://user-images.githubusercontent.com/91433976/170573517-23b228f9-2ef9-4b3e-9ac4-ae1fb0be1fe4.png)

5. Method DELETE:   
![image](https://user-images.githubusercontent.com/91433976/170573651-02c834d3-e55d-42cb-bf6e-c87c5bc51f74.png)
![image](https://user-images.githubusercontent.com/91433976/170573678-8837d6d6-f9a7-4c59-8642-be8bad33f230.png)

6. Method UPDATE:
   ![image](https://user-images.githubusercontent.com/91433976/170573873-a9a49c82-0795-4bbf-8e53-7acb909a9cdd.png)
   ![image](https://user-images.githubusercontent.com/91433976/170573907-39c5561c-5256-496b-8346-b4eb41b3536a.png)

7. Check if the component looks just like the mockup in different screen sizes

8. Read the changed files section on GitHub

9. Test the things you think are worth testing, even the ones that are not in this description
